### PR TITLE
Q&A image support 🏞️

### PR DESCRIPTION
**Pull request by addisonhernandez**
Usage:

From the `Add an Answer` modal, just paste a complete URL for an image anywhere on the internet!

You can also click on an answer thumbnail to see it biggified

### CODE REVIEW

**Review by Acid-Override — COMMENTED** [review]
Team Mountain Hackers!

**Review by Acid-Override — APPROVED** [review]

File: client/src/components/QuestionAnswers/Answer.jsx [hunk 1]
```diff
@@ -1,19 +1,27 @@
...
+import Modal from '../modals/Modal';
+import ImageModal from '../modals/ImageModal';
 
-  const {
+export default function Answer({
+  answer: {
     id,
     body,
-    answerer_name: answererName,
+    answerer_name: answerer,
```
Comment (Acid-Override) [inline client/src/components/QuestionAnswers/Answer.jsx:14]: 🧨

File: client/src/components/QuestionAnswers/Answer.jsx [hunk 2]
```diff
@@ -39,37 +47,78 @@ export default function Answer({ answer }) {
...
+  useEffect(() => {
+    if (photos && photos.length) {
+      setPhotosOrNah(
+        <>
+          {photos.map((url) => (
+            <Thumbnail
+              key={url}
+              src={url}
+              onClick={() => handleThumbnailClick(url)}
+              alt="An answer worth a thousand words"
```
Comment (Acid-Override) [inline client/src/components/QuestionAnswers/Answer.jsx:64]: 😄

**Comment by addisonhernandez** [comment]
Now I just need to render a new question or answer directly to the page. I think I've made that really difficult with my file structure. I may extend the Context so that I don't have to drill :sweat_smile:

**Comment by addisonhernandez** [comment]
[Add image support for Q&A answers](https://trello.com/c/q8gTWIUt/45-add-image-support-for-qa-answers)

File: client/src/components/QuestionAnswers/Answer.jsx (hunk 1 above)
Comment (addisonhernandez) [reply client/src/components/QuestionAnswers/Answer.jsx:14]: lol big changes here

**Pull request merged**